python/lsst/ip/diffim/imageDecorrelation.py

```diff
@@ -261,7 +274,7 @@ def run(self, scienceExposure, templateExposure, subtractedExposure, psfMatching
             corr = self.computeDiffimCorrection(kArr, varianceMean, targetVarianceMean)
 
         correctedImage = self.computeCorrectedImage(corr.corrft, subtractedExposure.image.array)
-        correctedPsf = self.computeCorrectedDiffimPsf(corr.corrft, psfImg.array)
```

Comment:
I'd just keep `psfImg` as `afw.Image` object. If you revert this change, then `computeAveragePsf` must return an object of that type.